src/components/SideNavBar/OldSideNavBar.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import style from './oldStyle.scss';
+
+class SideNavBar extends Component {
```

Comment:
Probably but this component will be removed once we make the switch over in Spacestation to the new sidenav.